public/js/profile.js

```diff
@@ -0,0 +1,68 @@
+/**
+* Submits the new updates for the profile
+*/
+$('#editForm').submit(function(evt) {
+    evt.preventDefault();
+
+    if ($('#passwd').val() === $('#confirmpasswd').val()) {
+        var id = $('#userId').html();
```

Comment:
We need to check that both password and confirm password are the same in the backend side as well as the api

Reply:
agree

Reply:
Yes, better to have backend validation as well, since front end can fail